### PR TITLE
[Layout] KBP V60 Type R Added ISO default

### PR DESCRIPTION
**Pull request by Lukewh**
## Description

Added the default UK ISO layout for KBP V60 Type R (KBP-V60R0B-UK)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* ¯\\\_(ツ)_/¯ - My desire to have a base config to riff off

## Checklist

- [x] My code follows the code style of this project. (I think so)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/v60_type_r/keymaps/iso/config.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+// place overrides here
+
+
+#ifdef V60_POLESTAR
```
Comment (mechmerlin) [inline keyboards/v60_type_r/keymaps/iso/config.h]: By using `#pragma once` you don't need the ifdef clauses.

Comment (mechmerlin) [inline keyboards/v60_type_r/keymaps/iso/config.h]:
```suggestion

```

File: keyboards/v60_type_r/keymaps/iso/config.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+#undef V60_POLESTAR
```
Comment (mechmerlin) [inline keyboards/v60_type_r/keymaps/iso/config.h]:
```suggestion

```

File: keyboards/v60_type_r/keymaps/iso/config.h [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+#endif
```
Comment (mechmerlin) [inline keyboards/v60_type_r/keymaps/iso/config.h]:
```suggestion

```

File: keyboards/v60_type_r/keymaps/iso/config.h (hunk 1 above)
Comment (Lukewh) [reply keyboards/v60_type_r/keymaps/iso/config.h]: Thanks :+1: I don't write C normally so still learning

Comment (Lukewh) [reply keyboards/v60_type_r/keymaps/iso/config.h]: I always forget about "Add suggestion to batch" but either way I've updated these now

File: keyboards/v60_type_r/info.json [hunk 4]
```diff
@@ -11,6 +11,9 @@
...
...w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"@", "x":11.75, "y":2}, {"label":"~", "x":12.75, "y":2}, {"label":"lshift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"rshift", "x":12.25, "y":3, "w":2.75}, {"label":"lctrl", "x":0, "y":4, "w":1.25}, {"label":"lwin", "x":1.25, "y":4, "w":1.25}, {"label":"lalt", "x":2.5, "y":4, "w":1.25}, {"label":"space", "x":3.75, "y":4, "w":6.25}, {"label":"fn0", "x":10, "y":4, "w":1.25}, {"label":"rwin", "x":11.25, "y":4, "w":1.25}, {"label":"menu", "x":12.5, "y":4, "w":1.25}, {"label":"rctrl", "x":13.75, "y":4, "w":1.25}]
```
Comment (noroadsleft) [inline keyboards/v60_type_r/info.json]: 1\. Reordering the JSON objects so Enter's object follows Non-US Hash's:

```suggestion
      "layout": [{"label":"esc", "x":0, "y":0}, {"label":"!", "x":1, "y":0}, {"label":"\"", "x":2, "y":0}, {"label":"\u00a3", "x":3, "y":0}, {"label":"$", "x":4, "y":0}, {"label":"%", "x":5, "y":0}, {"label":"^", "x":6, "y":0}, {"label":"&", "x":7, "y":0}, {"label":"*", "x":8, "y":0}, {"label":"(", "x":9, "y":0}, {"label":")", "x":10, "y":0}, {"label":"_", "x":11, "y":0}, {"label":"+", "x":12, "y":0}, {"label":"backspace", "x":13, "y":0, "w":2}, {"label":"tab", "x":0, "y":1, "w":1.5}, {"label":"Q", "x":1.5, "y":1}, {"label":"W", "x":2.5, "y":1}, {"label":"E", "x":3.5, "y":1}, {"label":"R", "x":4.5, "y":1}, {"label":"T", "x":5.5, "y":1}, {"label":"Y", "x":6.5, "y":1}, {"label":"U", "x":7.5, "y":1}, {"label":"I", "x":8.5, "y":1}, {"label":"O", "x":9.5, "y":1}, {"label":"P", "x":10.5, "y":1}, {"label":"{", "x":11.5, "y":1}, {"label":"}", "x":12.5, "y":1}, {"label":"caps", "x":0, "y":2, "w":1.75}, {"label":"A", "x":1.75, "y":2}, {"label":"S", "x":2.75, "y":2}, {"label":"D", "x":3.75, "y":2}, {"label":"F", "x":4.75, "y":2}, {"label":"G", "x":5.75, "y":2}, {"label":"H", "x":6.75, "y":2}, {"label":"J", "x":7.75, "y":2}, {"label":"K", "x":8.75, "y":2}, {"label":"L", "x":9.75, "y":2}, {"label":":", "x":10.75, "y":2}, {"label":"@", "x":11.75, "y":2}, {"label":"~", "x":12.75, "y":2}, {"label":"enter", "x":13.75, "y":1, "w":1.25, "h":2}, {"label":"lshift", "x":0, "y":3, "w":1.25}, {"label":"|", "x":1.25, "y":3}, {"label":"Z", "x":2.25, "y":3}, {"label":"X", "x":3.25, "y":3}, {"label":"C", "x":4.25, "y":3}, {"label":"V", "x":5.25, "y":3}, {"label":"B", "x":6.25, "y":3}, {"label":"N", "x":7.25, "y":3}, {"label":"M", "x":8.25, "y":3}, {"label":"<", "x":9.25, "y":3}, {"label":">", "x":10.25, "y":3}, {"label":"?", "x":11.25, "y":3}, {"label":"rshift", "x":12.25, "y":3, "w":2.75}, {"label":"lctrl", "x":0, "y":4, "w":1.25}, {"label":"lwin", "x":1.25, "y":4, "w":1.25}, {"label":"lalt", "x":2.5, "y":4, "w":1.25}, {"label":"space", "x":3.75, "y":4, "w":6.25}, {"label":"fn0", "x":10, "y":4, "w":1.25}, {"label":"rwin", "x":11.25, "y":4, "w":1.25}, {"label":"menu", "x":12.5, "y":4, "w":1.25}, {"label":"rctrl", "x":13.75, "y":4, "w":1.25}]
```

File: keyboards/v60_type_r/keymaps/iso/keymap.c [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+   * |Caps   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|  #|   |
+   * |-----------------------------------------------------------|
+   * |LShif|  ||  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /| RShift  |
+   * |-----------------------------------------------------------|
+   * |Ctrl|Gui |Alt |      Space              |Fn0 |Gui |App|Ctrl|
+   * `-----------------------------------------------------------'
+   */
+  [0] = LAYOUT_60_iso( \
+      KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, \
+      KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_ENT, \
```
Comment (noroadsleft) [inline keyboards/v60_type_r/keymaps/iso/keymap.c]: 2\. Moving `KC_ENT` to home row:

```suggestion
      KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, \
```

File: keyboards/v60_type_r/keymaps/iso/keymap.c [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+      KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOTE, KC_NUHS, \
```
Comment (noroadsleft) [inline keyboards/v60_type_r/keymaps/iso/keymap.c]:
```suggestion
      KC_CAPS, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOTE, KC_NUHS, KC_ENT, \
```

File: keyboards/v60_type_r/keymaps/iso/keymap.c [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+   * |       |lft|dwn|rig|   |   |   |   |hom|pgu|lft|rig|   |   |
+   * |-----------------------------------------------------------|
+   * |     |   |   |   |   |   |vld|vlu|mut|end|pgd|dwn|         |
+   * |-----------------------------------------------------------|
+   * |    |    |    |                         |    |    |   |    |
+   * `-----------------------------------------------------------'
+   */
+  [1] = LAYOUT_60_iso( \
+      KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_DEL, \
+      KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP, KC_TRNS, KC_TRNS, \
```
Comment (noroadsleft) [inline keyboards/v60_type_r/keymaps/iso/keymap.c]: 2a\. same change for Layer 1:

```suggestion
      KC_TRNS, KC_TRNS, KC_UP, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PSCR, KC_SLCK, KC_PAUS, KC_UP, KC_TRNS, \
```

File: keyboards/v60_type_r/keymaps/iso/keymap.c [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+      KC_TRNS, KC_LEFT, KC_DOWN, KC_RIGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_HOME, KC_PGUP, KC_LEFT, KC_RIGHT, KC_TRNS, \
```
Comment (noroadsleft) [inline keyboards/v60_type_r/keymaps/iso/keymap.c]:
```suggestion
      KC_TRNS, KC_LEFT, KC_DOWN, KC_RIGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_HOME, KC_PGUP, KC_LEFT, KC_RIGHT, KC_TRNS, KC_TRNS, \
```

File: keyboards/v60_type_r/v60_type_r.h [hunk 9]
```diff
@@ -105,3 +105,33 @@ enum my_keycodes {
...
+ * | K20     | K21 | K22 | K23 | K24 | K25 | K26 | K27 | K28 | K29 | K2A | K2B | K2C |       |
+ * |-----------------------------------------------------------------------------------------+
+ * | K30 | K31 | K32 | K33 | K34 | K35 | K36 | K37 | K38 | K39 | K3A | K3B | K3C             |
+ * |-----------------------------------------------------------------------------------------+
+ * | K40   | K41   | K42   | K43                             | K44   | K45   | K46   | K47   |
+ * `-----------------------------------------------------------------------------------------'
+ */
+#define LAYOUT_60_iso( \
+	       k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, \
+	       k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, \
```
Comment (noroadsleft) [inline keyboards/v60_type_r/v60_type_r.h]: 3\. And same change here:

```suggestion
	       k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, \
```

File: keyboards/v60_type_r/v60_type_r.h [hunk 10]
```diff
@@ -105,3 +105,33 @@ enum my_keycodes {
...
+	       k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, \
```
Comment (noroadsleft) [inline keyboards/v60_type_r/v60_type_r.h]:
```suggestion
	       k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k1d, \
```

Notice the same identifier deleted from the line above is added to this line, but the electrical matrix definition (Lines 129-136) is unchanged.

**Comment by noroadsleft** [comment]
@mechmerlin's review suggestions were implemented, but GitHub did not resolve the conversations. Marking manually.

File: keyboards/v60_type_r/info.json (hunk 4 above)
Comment (Lukewh) [reply keyboards/v60_type_r/info.json]: Thanks @noroadsleft, I'll make these changes locally to test them tomorrow than apply the suggestions if all is ok :).

**Pull request merged**